run-bahmni.sh

```diff
@@ -86,18 +94,25 @@ function showOpenMRSlogs {
 
 function startMart {
     echo "Starting services with profile 'bahmni-mart'..."
-    docker compose --profile bahmni-mart up -d
+    docker compose --env-file "$1" --profile bahmni-mart up -d
 }
 
 function pullLatestImages {
-    echo "Pulling all latest images..."
-    docker compose pull
+    if [ "$1" == ".env" ]; then
+        echo "Pulling all 1.0.0 images..."
+    elif [ "$1" == ".env.dev" ]; then
+        echo "Pulling all latest images..."
```

Comment:
We don't know this for sure. People can change .env or .env.dev to contain different images. Just provide a console message saying: Running with which .env file. That is all we know for sure.

Reply:
Updated the message
`echo "Pulling all the images specified in the $file file..."`